### PR TITLE
fix: [2U-704] provided the page title for new MFE studio pages (upstream)

### PR DESCRIPTION
**Pull request by PKulkoRaccoonGang**
### **Description**
Added the page title for new MFE studio pages:
- Grading page
- Advanced settings page
- Course team page
- Schedule and details page
- Course updates

[YouTrack](https://youtrack.raccoongang.com/issue/2U-704)

### CODE REVIEW

File: src/advanced-settings/__mocks__/advancedSettings.js [hunk 1]
```diff
@@ -13,4 +13,11 @@ module.exports = {
     hide_on_enabled_publisher: false,
     value: true,
   },
+  displayName: {
```
Comment (ruzniaievdm) [inline src/advanced-settings/__mocks__/advancedSettings.js:16]: please change naming to `camelCase` here and above

File: src/course-updates/hooks.jsx [hunk 2]
```diff
@@ -23,6 +24,7 @@ const useCourseUpdates = ({ courseId }) => {
   const [isUpdateFormOpen, openUpdateForm, closeUpdateForm] = useToggle(false);
   const [isDeleteModalOpen, openDeleteModal, closeDeleteModal] = useToggle(false);
   const [currentUpdate, setCurrentUpdate] = useState(initialUpdate);
+  const courseDetails = useModel('courseDetails', courseId);
```
Comment (ruzniaievdm) [inline src/course-updates/hooks.jsx:27]: optional `const { name: courseName } = useModel('courseDetails', courseId);`

File: src/course-updates/hooks.jsx [hunk 3]
```diff
@@ -93,6 +95,7 @@ const useCourseUpdates = ({ courseId }) => {
   }, [courseId]);
 
   return {
+    courseName: courseDetails?.name,
```
Comment (ruzniaievdm) [inline src/course-updates/hooks.jsx:98]: courseName,

**Review by ruzniaievdm — CHANGES_REQUESTED** [review]

File: src/advanced-settings/__mocks__/advancedSettings.js (hunk 1 above)
Comment (PKulkoRaccoonGang) [reply src/advanced-settings/__mocks__/advancedSettings.js:16]: Done, thanks

**Review by ruzniaievdm — APPROVED** [review]

**Review by vladislavkeblysh — APPROVED** [review]

**Comment by wowkalucky** [comment]
If it is intended to be "upstream", shouldn't it be targeted to external repo?

**Pull request closed without merging**